kuma/settings/common.py

```diff
@@ -1096,6 +1096,13 @@ def pipeline_one_scss(slug, **kwargs):
         ),
         'output_filename': 'build/js/selectivizr.js',
     },
+    'experiment-interactive-editor': {
+        'source_filenames': (
+            'js/libs/mozilla.trafficcop.js',
```

Comment:
I think you need additional libraries, unless these have been merged into ``mozilla.trafficcop.js``:

https://github.com/mozilla/kuma/blob/e780a32a1c31acbbe0d8597fea970605b0b39d05/kuma/settings/common.py#L1062-L1070